src/screens/EncounterHomeScreen.tsx

```diff
@@ -0,0 +1,105 @@
+import * as Linking from 'expo-linking';
+import React from 'react';
+import { ScrollView, StyleSheet, View } from 'react-native';
+import { Divider } from 'react-native-elements';
+import QRCode from 'react-native-qrcode-svg';
+
+import {
+  BoldText,
+  Button,
+  DiagonalGradient,
+  RegularText,
+  SectionHeader,
+  Wrapper,
+  WrapperRow,
+  WrapperWithOrientation
+} from '../components';
+import { LoadingSpinner } from '../components/LoadingSpinner';
+import { colors, device, Icon, normalize, texts } from '../config';
+import { useEncounterUser, useQRValue } from '../hooks';
+
+// TODO: accesibility labels
+export const EncounterHomeScreen = () => {
+  const { loading: loadingQr, qrId } = useQRValue();
+  const { loading: loadingUser, firstName, lastName, verified } = useEncounterUser();
+
+  const loading = loadingQr || loadingUser;
+
+  const qrValue = Linking.createURL('encounter', { queryParams: { qrId } });
+
+  if (loading) {
+    return <LoadingSpinner loading />;
+  }
+
+  return (
+    <ScrollView>
+      <SectionHeader title={texts.encounter.homeTitle} />
```

Comment:
i think we need the orientation wrapper more up to here, because otherwise the section header goes to wide

<img width="512" alt="Bildschirmfoto 2021-09-15 um 19 36 57" src="https://user-images.githubusercontent.com/1942953/133482245-ea44a2b6-80a8-41f3-bf29-94c94d2571ea.png">

Reply:
done in #330